implementations/ens/ipfs-contenthash/polywrap.deploy.yaml

```diff
@@ -7,3 +8,11 @@ jobs:
         uri: fs/./build
         config:
           gatewayUri: https://ipfs.wrappers.io
+      - name: http_deploy
+        package: http
+        uri: $$ipfs_deploy
+        config:
+          postUrl: https://wraps.wrapscan.io/r/wrap/ens-ipfs-contenthash-uri-resolver@1.0.0
```

Comment:
I'm confused, shouldn't the URL be `/r/polywrap/...` instead of `/r/wrap/...`?

Reply:
Woops, no idea how this happened